### DISCUSSION TITLE
after deploying express app, in vercel. my Index.js(root) code showing instead of working as a api like local machine. why  ?

### DISCUSSION THREAD

**Discussion by IAkashMondal** [vercel/community#7103] [Help]
### Summary

after deploying the express app, in Vercel. my Index.js(root) code showing instead of working as an
 API like my local machine. why?
 
![vercel2](https://github.com/vercel/community/assets/102036593/5c6a054c-17b9-47a4-976c-58a00ac4f524)
![vercel2](https://github.com/vercel/community/assets/102036593/70b7cde7-f848-43f7-859f-d9278d2249aa)
![vercel](https://github.com/vercel/community/assets/102036593/f1a584b0-650e-4f53-9e96-3800daaa6f1b)

repo link: https://github.com/IAkashMondal/DEV_CARFT/tree/master/06.MongoAtlas_Mvc

### Example

https://machines-eta.vercel.app/api/All

### Steps to Reproduce

for any future comunication reply below. thanks

**Comment by IAkashMondal** [comment]
also added this 
vercel.json :
1st:
{
  "version": 2,
  "builds": [{ "src": "./index.js", "use": "@vercel/node" }],
  "routes": [{ "src": "/(.*)", "dest": "/" }]
}
2nd:
{ "version": 2, "rewrites": [{ "source": "/(.*)", "destination": "/index.js" }] }

**Comment by leerob** [comment]
Could you try creating a smaller, reproducible example of the issue?

- https://vercel.com/guides/using-express-with-vercel
- https://vercel.com/guides/creating-a-minimal-reproducible-example
